### PR TITLE
interview questions and technical test feedback

### PR DESCRIPTION
**Pull request by thegsi**

### CODE REVIEW

**Comment by iteles** [comment]
Amazing, thanks @thegsi!

Is this ready to be merged?

File: README.md~ [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# interview-prep
```
Comment (nelsonic) [inline README.md~]: Is this a _duplicate_ file...?

**Comment by nelsonic** [comment]
@thegsi yes, this is _superb_! :+1:

File: README.md~ (hunk 1 above)
Comment (iteles) [inline README.md~]: Good point!

**Comment by hdrdavies** [comment]
Should be good now @nelsonic @iteles

**Comment by nelsonic** [comment]
Thanks for tidy up @hdrdavies 
Merging. :+1:

**Pull request merged**